### ISSUE TITLE
Different outputs with torchscript file between python and rust

### ISSUE THREAD

**Issue by yigitsoy** [LaurentMazare/tch-rs#487]
Hi,

I have been trying to use Rust for inference with a serialised model from python. For the same image (8-bit, grayscale), with the same preprocessings applied, I am getting different outputs.

Using the following python script, I get the `result tensor([[ 5.3778, -7.3396, -1.9843]], grad_fn=<AddmmBackward0>)`

```python
import torch

from monai.transforms import (
    Compose,
    LoadImage,
    NormalizeIntensity,
    ToTensor,
)


transforms = Compose([
            LoadImage(image_only=True), NormalizeIntensity(), ToTensor()
        ])
inp = transforms("test_tch.png")
scripted_model = torch.jit.load("model.pt")
scripted_model.eval()
output = scripted_model(inp.unsqueeze(0).unsqueeze(0))
print("result", output)
```

I expect the same result from the Rust counterpart, but the Rust code gives me `
-6.0697  5.9371 -3.8800
[ CPUFloatType{1,3} ]
`  (Note that `NormalizeIntensity` does a simple normalization with calculatated mean and standard deviation. )

```rust

use image::open;
use tch::{Kind, Tensor};

let im = open("test_tch.png").unwrap();
let sz = vec![ im.height() as i64, im.width() as i64];
let mut tensor: Tensor =  Tensor::of_data_size(im.into_bytes(), sz.as_slice(), Kind::Uint8)
        .unsqueeze(0)
        .unsqueeze(0);
tensor = tensor.to_kind(Kind::Float);
// normalize intensity
let (sd, mn) = tensor.std_mean(false);    
tensor = tensor.subtract(&mn).divide(&sd);
let mut model = tch::CModule::load("model.pt").unwrap();
model.set_eval();
let output = tensor.apply(&model);
output.print();
```

I have tested with a static import with `torch.ones` and both give the same result. I also verified that the image intensity statistics are the same after preprocessing (i.e. mean/std etc).

It would be great if you could give some hints on how to produce the same output from both scripts.

**Comment by yigitsoy** [comment]
Ok, it looks like a permutation of image axes is missing, which I had suspected when I got same results on static images.
